composer.json

```
@@ -49,7 +49,7 @@
         "twig/markdown-extra": "^3.3"
     },
     "require-dev": {
-        "dama/doctrine-test-bundle": "^8.0.0-beta5",
+        "dama/doctrine-test-bundle": "^8.0.0",
         "doctrine/doctrine-fixtures-bundle": "4.0.x-dev",
```

```suggestion
        "doctrine/doctrine-fixtures-bundle": "^3.5",
```
Same as https://github.com/symfony/demo/pull/1457/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R53

---

Why going back down to 3.5?

---

That's the latest stable